pkg/common/file.go

```diff
@@ -0,0 +1,70 @@
+package common
```

Comment:
I would avoid to use common package name. Often the purpose of package with poor names become difficult to understand. I would move this file inside client package in a file called homedir.go.